src/components/Datagrid/styles.ts

```diff
@@ -67,11 +69,11 @@ const useStyles = makeStyles(
           appearance: "none",
           background: "none",
           border: "none",
-          fontSize: themes.defaultLight.fontSize.bodySmall,
-          letterSpacing: "0.015em",
-          lineHeight: themes.defaultLight.lineHeight.bodySmall,
-          fontWeight: themes.defaultLight.fontWeight.bodySmall,
-          padding: themes.defaultLight.space[3],
+          fontSize: vars.fontSize.bodySmall,
```

Comment:
**suggestion**: I think it won't work with Datagrid

The problem with Datagrid is that it won't respect CSS variables e.g `--mu-background-plain` won't work but CSS values like `hsla(100%, 100%, 0,1)` will. That is the reason I used direct CSS values from the theme.

Reply:
It work because it applies for mui makeStyles not directly to datagrid.